server/s3_object_action_base.cc

```diff
@@ -106,11 +107,93 @@ void S3ObjectAction::fetch_object_info() {
 }
 
 void S3ObjectAction::fetch_object_info_success() {
-  request->set_object_size(object_metadata->get_content_length());
+  s3_log(S3_LOG_INFO, stripped_request_id, "%s Entry\n", __func__);
   next();
 }
 
-void S3ObjectAction::load_metadata() { fetch_bucket_info(); }
+void S3ObjectAction::fetch_additional_bucket_info() {
```

Comment:
Put comment here why this function in context of copy

Reply:
done